tests/baselines/wav2vec2_base.json

```diff
@@ -35,8 +35,8 @@
                     "learning_rate": 5e-4,
                     "train_batch_size": 32,
                     "eval_accuracy": 0.7829483695652174,
```

Comment:
Is the accuracy similar to what we get without HPU graphs? I think there was an issue with 1.12 but I have not tried it with 1.13.